thirdparty/bzip2/bzip2.patch

```diff
@@ -0,0 +1,31 @@
+diff -rupN orig/CMakeLists.txt patched/CMakeLists.txt
```

Comment:
A justification for this file (in a comment or sg) would be nice.

Reply:
bzip2 only provides a Makefile which is not very usable on Windows, and it is a pretty simple project so I created a CMake file to compile it on all platforms.